app/script/localization/webapp.js

```diff
@@ -605,6 +605,31 @@ z.string.preferencesOptionsCallLogs = 'Troubleshooting';
 z.string.preferencesOptionsCallLogsGet = 'Save the calling debug report';
 z.string.preferencesOptionsCallLogsDetail = 'This information helps Wire Support diagnose calling problems.';
 
+z.string.preferencesOptionsBackupHeader = 'History';
+z.string.preferencesOptionsBackupExportHeadline = 'Back up conversations';
+z.string.preferencesOptionsBackupExportSecondary = 'Create a backup to preserve your conversation history. You can use this to restore history if you lose your computer or switch to a new one.\nThe backup file is not protected by Wire end-to-end encryption, so store it in a safe place.';
+z.string.preferencesOptionsBackupImportHeadline = 'Restore from backup';
+z.string.preferencesOptionsBackupImportSecondary = 'You can only restore history from a backup of the same platform. Your backup will overwrite the conversations that you may have on this device.';
+
+z.string.backupErrorAction = 'Ok';
+z.string.backupErrorSecondaryAction = 'Cancel';
+z.string.backupExportGenericErrorHeadline = 'The file could not be saved';
+z.string.backupExportGenericErrorSecondary = 'You may not have enough space or permission to save it there.';
+z.string.backupExportProgressHeadline = 'Preparing…';
+z.string.backupExportProgressSecondary = 'Backing up · 7 of 10 MB — 70%';
```

Comment:
Needs placeholders for `7`, `10 MB` and `70%`

Reply:
already done on my side